plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java

```diff
@@ -247,6 +248,31 @@ public void testLimitPushdown()
         abort("TODO");
     }
 
+    @Test
+    @Disabled
+    @Override
+    public void testCommentTable()
+    {
+        // This connector does not support setting table comments
+    }
+
+    @Test
+    @Override
+    public void testCommentTableSpecialCharacter()
+    {
+        assertThatThrownBy(super::testCommentTableSpecialCharacter)
+                .hasStackTraceContaining("This connector does not support setting table comments");
+    }
+
+    @Test
+    @Override
+    public void testDataMappingSmokeTest()
+    {
+        // TODO: Connector does not support type timestamptz for the first commit
+        assertThatThrownBy(super::testDataMappingSmokeTest)
+                .hasStackTraceContaining("Unsupported column type(2014):timestamptz");
```

Comment:
Please update `filterDataMappingSmokeTestData` instead.

Reply:
After trying to block it by filterDataMappingSmokeTestData, it seems that it doesn't work well. I always get the error message for timestamptz. So I picked our handle back for timestamptz. And it works now.